src/theme.c

```diff
@@ -128,6 +128,9 @@ theme_builtin(struct theme *theme)
 	parse_hexstr("#dddad6", theme->menu_items_active_bg_color);
 	parse_hexstr("#000000", theme->menu_items_active_text_color);
 
+	theme->menu_min_width = 20;
+	theme->menu_max_width = 200;
```

Comment:
Opinions for default values? I chose 20 for min here because that allows to have a menu consisting of only single emojis as menu entries. Although that likely depends on the font size as well..

Reply:
Fine.